chia/rpc/wallet_rpc_client.py

```diff
@@ -238,10 +238,12 @@ async def cat_asset_id_to_name(self, asset_id: bytes32) -> Optional[Tuple[uint32
         }
         try:
             res = await self.fetch("cat_asset_id_to_name", request)
-            return uint32(int(res["wallet_id"])), res["name"]
-        except Exception:
+        except ValueError:
             return None
 
+        wallet_id: Optional[uint32] =  None if res["wallet_id"] is None else uint32(int(res["wallet_id"]))
+        return wallet_id, res["name"]
```

Comment:
This type doesn't look right. This function returns an optional tuple, but you're returning a tuple with an optional here

Reply:
Yeah due to this patch that is now optional because you may get an asset ID that is not associated with a wallet of yours.  I'll update the function return type to reflect that.